examples/src/main/scala/example/ziohttp/ExampleApp.scala

```diff
@@ -0,0 +1,26 @@
+package example.zhttp
+
+import example.ExampleData._
+import example.ExampleService.ExampleService
+import example.{ ExampleApi, ExampleService }
+
+import zio._
+import zhttp._
+import zhttp.http._
+import caliban.ZHTTPAdapter
+
+object ExampleApp extends App {
+  override def run(args: List[String]): ZIO[ZEnv, Nothing, ExitCode] = {
```

Comment:
This will be the simplest example of all adapters =)
Can you add `graphiql` so that it's same as the others?

Reply:
Absolutely!

Reply:
Fixed in 6dc046f.